setup.py

```diff
@@ -26,6 +26,8 @@
         "scipy",
         "huggingface-hub>=0.15.1",
         "Pillow",
+        "datasets",
+        "accelerate",
```

Comment:
Same nit here for the accelerate version

Reply:
Addressed with a0071a57d7f659233d9d8d5a6bacd01acc7971b1. Thanks!